documentation/user_docs/docs/manual/Manipulating_and_extracting_data_from_SQW_files_and_objects.rst

```diff
@@ -2,166 +2,367 @@
+.. warning::
 
+   Plots that are produced plot them as orthogonal axes so any features may be skewed.
 
-- ``'-nopix'`` means that the individual pixel information contributing to the resulting data is NOT retained (at present the default is to retain it, resulting in an output that is an sqw object, whereas using ``'-nopix'`` gives a dnd output).
-- ``filename`` is a string specifying a full filename (including path) for the data to be stored, in addition to being stored in the Matlab workspace.
+   However, it does make reading the location of a feature in a two-dimensional **Q**-**Q** plot straightforward, which is the main reason for doing this.
 
-cut
-===
+Spherical Projections
+---------------------
+
+TBD
+
+Cylindrical Projections
+-----------------------
+
+TBD
+
+Binning arguments
+-----------------
+
+* ``p1_bin``, ``p2_bin``, ``p3_bin`` and ``p4_bin``
+  specify the binning / integration arguments for the Q & Energy axes in
+  the target coordinate system. Each can independently have one of four different forms:
+
+.. warning::
+
+   The meaning of the first, second, third, etc. components changes between each form. Ensure that you have the correct
+   value in each component to ensure your cut is what you expect.
+
+* ``[]``
+  An empty binning range will use the current/previous binning axes in that dimension.
+
+* ``[n]``
+  if a single (scalar) number is given then that axis will be a plot axis and the bin width will be the
+  number you specify. The lower and upper limits are the (current|previous) binning axes in that dimension.
+
+.. note::
+
+   A value of ``[0]`` is equivalent to ``[]`` and will use the (current|previous) binning axes.
+
+* ``[lo,hi]``
+  If you specify a vector with two components then the signal will be integrated over that axis between limits
+  specified by the two components of the vector.
+
+.. warning::
+
+   A two-component binning axis defines the integration region between bin edges. For example, ``[-1 1]`` will capture
+   pixels from ``-1`` to ``1`` inclusive.
+
+* ``[lower,step,upper]``
+
+  A three-component binning axis specifies an axis is a plot axis with the first ``lower`` and the last ``upper``
+  components specifying the centres of the first and the last bins of the data to be cut. The middle component specifies
+  the bin width.
+
+.. note ::
+
+   If ``step`` is ``0``, the ``step`` is taken from the (current|previous) binning axes.
+
+.. warning::
+
+   A three-component binning axis defines the integration region by bin centres, i.e. the limits of the
+   data to be cut lie between ``min = lower-step/2`` and ``max = upper+step/2``, including ``min/max``
+   values. For example, ``[-1 1 1]`` will capture pixels from ``-1.5`` to ``1.5`` inclusive.
+
+
+* ``[lower, separation, upper, cut_width]``
+
+  A four-component binning axis defines multiple cuts with multiple integration limits in the selected direction.
```

Comment:
going to suggest that multiple is heavily emphasized, bold type and stars fore and aft.